resources/views/listing/partials/filter/selected.blade.php

```diff
@@ -1,22 +1,24 @@
 <ais-clear-refinements>
     <template v-slot="{ canRefine, refine, createURL }">
-        <a
-            v-if="canRefine"
-            v-bind:href="createURL()"
-            v-on:click.prevent="refine"
-        >
-            @lang('Reset filters')
-        </a>
-
-        {{-- TODO: Without this the default renders... --}}
-        <div v-else></div>
+        <div v-show="canRefine" class="flex flex-wrap items-baseline justify-between gap-2 w-full pb-2">
+            <div class="font-semibold text-base">
+                @lang('You have filtered for:')
```

Comment:
I can't put my finger on it, i feel like this reads a bit weird. 
```suggestion
                @lang('Selected filters:')
```
or
```suggestion
                @lang('Current filters:')
```
may read a bit nicer

Reply:
Looks like we overwrite this in every single project to be without this label, and just show "reset filters".

But, if I were to choose, I'd pick "Selected filters:" here.